code/modules/research/xenoarchaeology/tools/bunsen_burner.dm

```diff
@@ -49,24 +48,39 @@
 		held_container = null
 	..()
 
+/obj/machinery/bunsen_burner/examine(mob/user)
+	..()
+	switch(heating)
+		if(1)
+			to_chat(user, "<span class = 'notice'>\The [src] is on.</span>")
+		if(-1)
+			to_chat(user, "<span class = 'notice'>\The [src]'s fuel port is open.</span>")
+	to_chat(user, "<span class='info'>It contains:</span>")
+	if(reagents && reagents.reagent_list.len)
+		for(var/datum/reagent/R in reagents.reagent_list)
+			to_chat(user, "<span class='info'>[R.volume] units of [R.name]</span>")
+	else
+		to_chat(user, "<span class='info'>Nothing.</span>")
+	if(held_container)
+		to_chat(user, "<span class='info'>It is holding a:</span>")
+		held_container.examine(user)
+
 /obj/machinery/bunsen_burner/attackby(obj/item/weapon/W, mob/user)
 	if(istype(W, /obj/item/weapon/reagent_containers))
 		var/obj/item/weapon/reagent_containers/R = W
-		if(held_container)
+		if(heating == -1)
```

Comment:
Now you've added `-1` it's becoming a bit of a mess. Use defines to clear it up.